resources/docs/livewire/showing-chirps.md

```diff
@@ -0,0 +1,313 @@
+[TOC]
+
+# <b>04.</b> Showing Chirps
+
+In the previous step we added the ability to create Chirps, now we're ready to display them!
+
+## Retrieving the Chirps
+
+So, first, we should update our `app/resources/views/chirps.blade.php` view to display a listing of Chirps. For that, we can use a new Livewire component:
+
+```blade filename=resources/views/chirps.blade.php
+<x-app-layout>
+    <div class="max-w-2xl mx-auto p-4 sm:p-6 lg:p-8">
+        <livewire:chirps.create />
+        <!-- [tl! add:start] -->
+        <livewire:chirps.list /><!-- [tl! add:end] -->
+    </div>
+</x-app-layout>
+```
+
+After, let's create a new `chirps.list` Livewire component:
+
+```shell tab=Class
+php artisan make:volt chirps/list --class
+```
+
+```shell tab=Functional
+php artisan make:volt chirps/list
+```
+
+This will create a new Livewire component at the `app/resources/views/livewire/chirps/list.blade.php` path. Let's update the Livewire component contents to display our list of Chirps:
```

Comment:
```suggestion
This will create a new Livewire component at the `resources/views/livewire/chirps/list.blade.php` path. Let's update the Livewire component contents to display our list of Chirps:
```

Reply:
Fixed.